lib/kafo/hook_context.rb

```diff
@@ -61,6 +61,13 @@ def add_module(module_name, mapping = nil)
       self.kafo.add_module(module_name)
     end
 
+    # Check if a module is enabled in the current configuration.
+    # examples:
+    #   module_enabled?('example')
+    def module_enabled?(module_name)
+      self.kafo.module(module_name).enabled?
```

Comment:
I think you can use [this](https://github.com/domcleal/kafo/blob/e572fe236f19316f7ab97b12ac65d39831e3743d/lib/kafo/configuration.rb#L169) directly